install-dev/classes/controllerHttp.php

```diff
@@ -375,7 +375,7 @@ public function getDocumentationLink()
     public function getTutorialLink()
     {
         /* Link to localized video tutorial (if available) */
-        return $this->translator->trans('https://www.youtube.com/watch?v=psz4aIPZZuk', array(), 'Install');
+        return $this->translator->trans('https://www.youtube.com/watch?v=cANFwuJqdgM', array(), 'Install');
```

Comment:
why this string is translated? :rofl:

Reply:
Because we can have some introduction of PrestaShop in many languages!

https://www.youtube.com/watch?v=psz4aIPZZuk => EN
https://www.youtube.com/watch?v=IOM9AffOa3s => FR
https://www.youtube.com/watch?v=OdwJwCZyKLI => ES

:smile: